### PR TITLE
Adding optional state/command suffix support

### LINKED ISSUE #25: Status Topic
I have been using this and it is incredible!  Thank you very much for putting this together for us SmartThings users who like to make our own interfaces in Home Assistant.

Would it be possible to include a separate state topic for each exposed ST device?

For example, we currently get events AND send commands to smartthings/Fireplace Lights/switch.
I'd like to be able to get events at smartthings/Fireplace Lights/switch/state and send commands to smartthings/Fireplace Lights/switch/cmd.  That way when I send a command, I can confirm that it worked by checking the state again.

Looking forward to your response!

### PR DESCRIPTION
**Pull request by stjohnjohnson**
This adds support for state and command suffixes.  But these are optional.

Fixes #25  @jer

### CODE REVIEW

**Comment by stjohnjohnson** [comment]
@jer please review

File: server.js [hunk 1]
```diff
@@ -183,15 +200,30 @@ function handleSubscribeEvent (req, res) {
...
+        suffix = config.mqtt[SUFFIX_COMMAND];
+    } else if (type === TOPIC_STATE) {
+        suffix = config.mqtt[SUFFIX_STATE];
+    }
+
+    if (suffix) {
+        tree.push(suffix);
+    }
+
+    return tree.join('/');
```
Comment (chilicheech) [inline server.js]: Looks like the default suffixes are empty strings. This line seems to add a trailing slash if you use the default suffixes. Is that the case and If so, will this be a problem for the topic's path?

Comment (stjohnjohnson) [inline server.js]: See the if statement above.  It only adds it to the tree if suffix is non-empty.

**Comment by jer** [comment]
Ok sure :+1:

Not entirely certain why this is useful though

**Pull request merged**